### ISSUE TITLE
Add rendering for amenity=hunting_stand

### ISSUE THREAD

**Issue by zdila** [gravitystorm/openstreetmap-carto#966]
Please use http://goo.gl/I6e5XW for amenity=hunting_stand.

I am the author of the icon and I am releasing it under public domain.

If you need to do any modification to it, please feel free or contact me.

**Comment by matkoniecz** [comment]
https://github.com/nebulon42/osmic/issues/7#issuecomment-71387781 has a nice icon.

For that feature I think that rendering is reasonable, especially as it appearing in places with a low feature density.

**Comment by nebulon42** [comment]
As a special service here is a version without halo :)
https://gist.githubusercontent.com/nebulon42/8ed48d5043fc01ea8987/raw/f05bec9825d86219019124e0576ec3e15b0f303f/hunting-stand-16.svg

**Comment by gabischatz** [comment]
https://community.openstreetmap.org/t/anzeige-von-hochsitzen-im-wald/144590
Hello,

I would like to raise a concern about the rendering of `amenity=hunting_stand` in the standard OpenStreetMap Carto style.

My point is not that these objects should be removed from OpenStreetMap. If they are physically present and verifiable, they may be legitimate map data.

However, in discussions with forestry workers and people responsible for forest areas, I repeatedly hear the concern that publicly and prominently displaying hunting stands can cause practical problems. Some of them report that hunting stands may be misused, visited by people who should not use them, or even damaged by people opposed to hunting.

For that reason, the question is not only whether hunting stands are mappable, but whether they should be shown prominently in the default map style used by many ordinary users.

Would it be possible to reconsider the rendering of `amenity=hunting_stand`, for example by showing it only at higher zoom levels, making it less prominent, or not rendering it in the default Carto style at all?

This would not delete the data from OSM. Specialized maps could still use the information, but the default map would avoid drawing unnecessary attention to objects that may be sensitive in practice.

Thank you for considering this.

**Comment by imagico** [comment]
First: This issue was about adding rendering of `amenity=hunting_stand`.  Removing or modifying its display would be a new issue, especially since the considerations presented are completely unrelated to the original discussion in 2015.

If you would like to suggest changing the rendering or completely removing it you would need to present arguments why that is a good idea under [the goals we have](https://github.com/openstreetmap-carto/openstreetmap-carto/blob/master/CARTOGRAPHY.md).

Under these goals the main arguments for removing rendering would be

* if the tag is not used consistently any more or it is not consensus tagging among mappers for something specific any more.  This is quite clearly not the case here.
* the display of the features in question is determined not to be useful for our target map users any more.  This is quite clearly not the case either.
* the design we have chosen is considered to be non-intuitive or misleading and no better alternative ideas exist.  Does not seem to be the case here either.
* the display of the features in question negatively affects the readability of the map otherwise in a way that is not warranted by the benefits.  Even at z16 and in areas with a high density of hunting stands this seems rarely the case.

In addition i would like to comment on the idea that seems to be presented in the subtext here that we might want to explicitly refrain from displaying information that is readily publicly available (both in OSM and on the ground) because some people based on their specific interest deem it undesirable for this information to be more broadly known and available to the general public.  With the increasing tendencies towards authoritarian political ideas throughout the global West this is likely to become an increasingly common request for OSM-Carto.

Speaking only for myself here:  I consider this not a suitable basis for design decisions in OSM-Carto.  The only arguments i can see potentially playing in beyond the question what benefits our goals and our cartographic strategy are *ethical arguments*.  And even there we would have to be very careful since superficially ethical considerations are often just a vehicle to pursue cultural imperialism.  For a valid ethical argument you should IMO be able to demonstrate that it is of net benefit of society at large to not communicate the information in question (utilitarian viewpoint) and that doing so follows an overall principle that is suitable for universal application (deontological viewpoint).  I have used such argument in a few cases (in particular for the display of casinos).  In most cases, however, this is a discussion that mappers will need to have regarding what they consider suitable for mapping.  The cases where something can be considered ethical to map but not ethical to display in a map are very limited in my eyes.

Bottom line: If, based on these considerations, you think you have a good case for suggesting some specific change about the rendering of `amenity=hunting_stand` then please open a new issue with that suggestion and present your arguments.
